src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -266,17 +266,19 @@ export default class ClinicalData extends React.Component<
                         })
                 );
 
-                let clinicalData = await client.fetchClinicalDataUsingPOST({
-                    clinicalDataType: attribute.patientAttribute
-                        ? 'PATIENT'
-                        : 'SAMPLE',
-                    clinicalDataMultiStudyFilter: {
-                        attributeIds: [attribute.clinicalAttributeId],
-                        identifiers: attribute.patientAttribute
-                            ? patientidentifiers
-                            : sampleIdentifiers,
-                    },
-                });
+                let clinicalData = await client
+                    .handleErrorsGlobally()
```

Comment:
this is an example of setting error handling at invocation time

Reply:
I like it